source/braille.py

```diff
@@ -1124,11 +1177,50 @@ def getFormatFieldBraille(field, fieldCache, isAtStart, formatConfig):
 				# Translators: brailled when text contains a bookmark
 				text = _("bkmk")
 				textList.append(text)
+
+	if (
+		config.conf["braille"]["fontAttributeDisplay"].calculated() == FontAttributesBrailleModeFlag.TAGS
+		and (formattingTags := _getFormattingTags(field, fieldCache, formatConfig)) is not None
+	):
+		textList.append(formattingTags)
+
 	fieldCache.clear()
 	fieldCache.update(field)
 	return TEXT_SEPARATOR.join([x for x in textList if x])
 
 
+def _getFormattingTags(
+	field: dict[str, str], fieldCache: dict[str, str], formatConfig: dict[str, bool]
+) -> str | None:
+	"""
+	Get the formatting tags for the given field and cache.
+
+	Args:
+		field: The format field.
+		fieldCache (dict): The previous format field.
+		formatConfig: The user's format config.
+
+	Returns:
+		The formatting tag as a string, or None if no formatting is applied.
+	"""
+
+	def appendFormatSymbol(attribute: str, startSymbol: str, endSymbol: str) -> None:
+		newVal = field.get(attribute, False)
+		oldVal = fieldCache.get(attribute, False) if fieldCache is not None else False
+		if newVal and not oldVal:
+			textList.append(startSymbol)
+		elif oldVal and not newVal:
+			textList.append(endSymbol)
```

Comment:
can this be moved out of this function?

Reply:
I nested it as it's only really useful here. Is there a good reason to un nest it?

Reply:
from the zen of python (subjective of course)

>Flat is better than nested. 
[more info](https://realpython.com/zen-of-python/#:~:text=Another%20pair%20of%20guidelines%20from,Sparse%20is%20better%20than%20dense.&text=This%20code%20is%20slightly%20more,it's%20also%20nested%20and%20verbose.)

TLDR: reduces function complexity, improves modularity and testability